src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -2290,5 +2290,70 @@ public static class C
             var comp = CreateCompilationWithMscorlib(source);
             CompileAndVerify(comp);
         }
+
+        [Fact, WorkItem(278481, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=278481&fullScreen=false&_a=edit")]
```

Comment:
I recommend a URI of the form `https://devdiv.visualstudio.com/DevDiv/_workitems?id=278481`